core/bcast/bcast_test.go

```diff
@@ -86,3 +87,40 @@ func TestBroadcastBeaconBlock(t *testing.T) {
 
 	<-ctx.Done()
 }
+
+func TestBroadcastVoluntaryExit(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	mock, err := beaconmock.New()
+	require.NoError(t, err)
+
+	ve := &eth2p0.SignedVoluntaryExit{
+		Message: &eth2p0.VoluntaryExit{
+			Epoch:          10,
+			ValidatorIndex: 10,
+		},
+		Signature: testutil.RandomEth2Signature(),
+	}
```

Comment:
you could extract this as `testutil.RandomSignedVoluntaryExit`

Reply:
i think it is too small to extract plus it would be used only here

Reply:
Done